app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -69,8 +79,20 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                     ])
                     ->get()
                     ->toArray(),
+                'articleVote' => ArticleWasHelpful::active() ? optional(
+                    $article->knowledgeBaseArticleVotes()
+                        ->where('user_id', $userid)
+                        ->where('user_type', $userType)
+                        ->select([
+                            'id',
+                            'is_helpful',
+                        ])
+                        ->first()
+                )->toArray() : [],
             ]),
             'portal_view_count' => $article->portal_view_count,
+            'guest_id' => Session::get('guest_id'),
+            'feature_flag' => ArticleWasHelpful::active(),
```

Comment:
To not conflict or be confusing with other feature flags I would rename this to `article_was_helpful_feature_flag`

Reply:
Done